pkg/reconciler/knativeeventing/common/sinkbindingselectionmode.go

```diff
@@ -73,3 +77,21 @@ func SinkBindingSelectionModeTransform(instance *eventingv1beta1.KnativeEventing
 		return nil
 	}
 }
+
+func sinkBindingSelectionModeFromWorkloadOverrides(instance *eventingv1beta1.KnativeEventing) string {
+	for _, workloadOverride := range instance.Spec.Workloads {
+		if workloadOverride.Name == "eventing-webhook" {
```

Comment:
Do you really need this configuration for `eventing-webhook` only?

Reply:
As I understood, only the eventing-webhook deployment has this env var (in this it is relevant). But that's probably something @pierDipi can say for sure.

Reply:
yeah, that's from the webhook, see: https://github.com/knative/eventing/search?q=SINK_BINDING_SELECTION_MODE


That said, we do however override the actual upstream default on our own opertor's extension: 
https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/pkg/eventing/extension.go#L66-L69


Could we have some test for that? (there is some extension code and tests in this repo)

Reply:
This change only enables the support in "instance.Spec.Workloads".
The resource "eventing-webhook" is a deployment. 
What if the user configure the env var in "instance.Spec.Deployments"?

Reply:
:+1: addressed in 681c3ad0